rero_ils/modules/documents_items/static/scss/rero_ils/thumbnail.scss

```diff
@@ -19,13 +16,13 @@
 }
 
 .icon-document {
-  border: 1px solid $gray-lighter;
+  border: 1px solid $light;
```

Comment:
Boostrap utilities can be used: https://getbootstrap.com/docs/4.0/utilities/borders/#border-color

Reply:
That's right, but not with this color. `border border-light` is like `border border-white`, and is like no border in our case.